railties/test/application/middleware/session_test.rb

```diff
@@ -339,6 +339,9 @@ def read_raw_cookie
 
         # Enable AEAD cookies
         config.action_dispatch.use_authenticated_cookie_encryption = true
+
+        # Enable Authenticated Message Encryption.
+        config.active_support.use_authenticated_message_encryption = true
```

Comment:
This will be true by default, so there's no need to set this.

We should assign the cbc `cipher:` to the legacy encryptor in cookies middleware. Then the test passes.